src/OnePlusBot/Modules/News.cs

```diff
@@ -1,44 +1,38 @@
 ﻿using System.Threading.Tasks;
-using Discord;
 using System;
 using Discord.Commands;
 using System.Linq;
 using Discord.WebSocket;
 using OnePlusBot.Base;
-using OnePlusBot.Helpers;
 
 namespace OnePlusBot.Modules
 {
     public class NewsModule : ModuleBase<SocketCommandContext>
     {
         [Command("news")]
         [Summary("Posts a News article to the server.")]
-        public async Task NewsAsync([Remainder] string news)
+        public async Task<RuntimeResult> NewsAsync([Remainder] string news)
         {
             var guild = Context.Guild;
 
+            var user = (SocketGuildUser)Context.Message.Author;
             var newsChannel = guild.GetTextChannel(Global.Channels["news"]);
             var newsRole = guild.GetRole(Global.Roles["news"]);
             var journalistRole = guild.GetRole(Global.Roles["journalist"]);
 
             if (news.Contains("@everyone") || news.Contains("@here") || news.Contains("@news")) 
-            {
-                await Context.Channel.EmbedAsync(new EmbedBuilder()
-                    .WithColor(9896005)
-                    .WithDescription("⚠ That news contains an illegal ping!"));
-                return;
-            }
+                return CustomResult.FromError("Do not use illegal pings!");
 
-            var user = (SocketGuildUser) Context.Message.Author;
 
             if (!user.Roles.Contains(journalistRole))
-                return;
+                return CustomResult.FromError("Jounralists only!");
```

Comment:
Improve error message to something like: `Only user with Journalists role are allowed to use this command`